.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ add_known_hosts: &add_known_hosts
 ## Defines images and working directory.
 defaults: &defaults
   docker:
-    - image: pookmish/drupal8ci:latest
+    - image: callinmullaney/drupalci:0.0.1
```

Comment:
Changes in this file should not be merged and are temporary. The docker image I built forks Mike's image adds Node v16.